src/pages/articles/python/converting-integer-to-string-in-python/index.md

```diff
@@ -3,13 +3,32 @@ title: Converting Integer to String in Python
 ---
 ## Converting Integer to String in Python
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/articles/python/converting-integer-to-string-in-python/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Unlike many other languages out there, Python does not implicitly typecast integers (or floats) to strings when concatenating with strings. Fortunately, Python has a handy built-in function `str()` which will convert the argument passed in to a string format.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+Simple concatenation example:
+```py
```

Comment:
Could you give an example where this would fail, and the resulting error message? Perhaps using the range example.

Then show the correct way with the range example, finishing off with your name/age example.